From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Kenneth Myhra <kennethmyhra@gmail.com>
Date: Tue, 18 Jul 2023 19:18:27 +0200
Subject: [PATCH] Add serenity alias to linker version script

This adds a serenity alias to the linker version script and forwards it
to the solaris configuration.
---
 util/mkdef.pl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/util/mkdef.pl b/util/mkdef.pl
index d9534674cde1ed722cbf665692b9efc541f1d01a..28976aafe65b97dfd6bab7c460cc0f9648d33ebd 100755
--- a/util/mkdef.pl
+++ b/util/mkdef.pl
@@ -113,6 +113,7 @@ my %OS_data = (
                      platforms  => { UNIX                       => 1 } },
     "solaris-gcc" => 'solaris', # alias
     linux       => 'solaris',   # alias
+    serenity    => 'solaris',   # alias
     "bsd-gcc"   => 'solaris',   # alias
     aix         => { writer     => \&writer_aix,
                      sort       => sorter_unix(),
